sentry-samples/sentry-samples-android/CMakeLists.txt

```diff
@@ -3,11 +3,12 @@ project(Sentry-Sample LANGUAGES C CXX)
 
 add_library(native-sample SHARED src/main/cpp/native-sample.cpp)
 
-add_subdirectory(../../sentry-android-ndk/${SENTRY_NATIVE_SRC} sentry_build)
+#add_subdirectory(../../sentry-android-ndk/${SENTRY_NATIVE_SRC} sentry_build)
+find_package(sentry-android REQUIRED CONFIG)
```

Comment:
this fails:
https://cmake.org/cmake/help/latest/command/find_package.html#find-package

>     Could not find a package configuration file provided by "sentry-android"
>     with any of the following names:
>   
>       sentry-androidConfig.cmake
>       sentry-android-config.cmake
>   
>     Add the installation prefix of "sentry-android" to CMAKE_PREFIX_PATH or set
>     "sentry-android_DIR" to a directory containing one of the above files.  If
>     "sentry-android" provides a separate development package or SDK, be sure it
>     has been installed.
>

Reply:
either sentry-android-ndk is not configured correctly with prefab or its an error here on the cmakelists